src/javascript/campaign-url-builder/components/campaign-url.js

```diff
@@ -224,27 +317,51 @@ export default class CampaignUrl extends React.Component {
    * @param {Object} nextProps
    */
   componentWillReceiveProps(nextProps) {
-    if (nextProps.url != this.props.url) {
+    if (nextProps.url !== this.props.url) {
       AlertDispatcher.removeAll();
       this.setState({
         shortUrl: null,
         showShortUrl: false,
         isShorteningUrl: false,
+        problematicBypass: false,
+      });
+
+      // Compute the renderProblematic, and dispatch a GA event if the
+      // state went from Not Problematic to Problematic
+      // TODO(nathanwest): deduplicate this and the constructor code
+      const {element, eventLabel} = renderProblematic(nextProps.url);
+      this.setState(prevState => {
+        if (isNil(prevState.problematicElement) && !isNil(element)) {
+          gaSendEvent({
+            category: 'Campaign URL',
+            action: 'Problematic URL',
```

Comment:
I don't understand what you mean by action like– you mean, the tag "Problematic URL" should be a verb?

Reply:
I mean the dimension name is "Event Action" and "Problematic URL" doesn't sound like an action to me. I was also saying it's probably worth taking a look at the other event actions we collect since whatever you choose, it's going to be in your reports forever.